packages/ckb-sdk-utils/src/crypto/blake2b.ts

```diff
@@ -374,10 +375,24 @@ export const blake2b = (
   return new Blake2b(outlen, key, salt, personal)
 }
 
-export const ready = (cb: Function) => {
-  b2wasm.ready(() => {
-    cb()
-  })
+export const ready = (cb: Function) => {  
+  const ua = navigator.userAgent.toLowerCase()
+  const IOSVersion = ua.match(/cpu iphone os (.*?) like mac os/)
```

Comment:
variable name should start with a lower case.

Reply:
naming conventions is updated